apps/api/src/domain/models/vocabulary-list/commands/create-vocabulary-list/vocabulary-list-created.event-handler.integration.spec.ts

```diff
@@ -105,6 +101,19 @@ describe(`VocabularyListCreatedEventHandler`, () => {
             expect(foundTextForName).toBe(vocabularyListName);
 
             expect(foundLanguageCodeForName).toBe(originalLanguageCode);
+
+            expect(view.actions).toContain('PUBLISH_RESOURCE');
```

Comment:
Nice!

Reply:
Yea, but I'm not 100% sold on this pattern. Instead of caching command types on the view documents, I think we might want to do
``` ts
const userActions =  viewModel.getAvailableCommands().map( commandType =>this.commandFinderService.buildForm(viewModel,commandType))
```